src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,38 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const statuses: Status[] = ['all', 'active', 'completed'];
```

Comment:
Looks like this constant should be moved outside component (or even to separate file)